src/workers/cdc.ts

```diff
@@ -165,6 +173,19 @@ const onUserChange = async (
     ) {
       await notifyDevCardEligible(logger, data.payload.after.id);
     }
+
+    if (
+      data.payload.after.reputation >= submissionAccessThreshold &&
+      data.payload.before.reputation < submissionAccessThreshold
```

Comment:
```suggestion
      data.payload.before.reputation <= submissionAccessThreshold
```

In my test I was using a reputation of 20, threshold of 21.
Then when updating it would not accept it as my before was not matching.

Seeing we are going to implement retro-active we should consider people being on exactly 250 already.

Also what will happen if they already on reputation above the threshold?
They will never be able to pass this.

Reply:
> In my test I was using a reputation of 20, threshold of 21.
Then when updating it would not accept it as my before was not matching.

I think the current condition is correct. If the threshold is 20 and your previous reputation is already 20, then you already passed before. Only jumping from 19 to 20 should make it triggered.

> Also what will happen if they already on reputation above the threshold?

We would only send emails to those who did not have access yet and then passed afterward. I think this is a great question if we will send emails to those who already passed on launch. WDYT? @idoshamun

Reply:
I agree with @sshanzel. I think the current condition is right. We will manually send emails to those who already qualified on launch.